platform/ios/src/MGLMapView.mm

```diff
@@ -663,48 +675,6 @@ - (void)reachabilityChanged:(NSNotification *)notification
     _isWaitingForRedundantReachableNotification = NO;
 }
 
-- (void)willMoveToWindow:(UIWindow *)newWindow
-{
-    [super willMoveToWindow:newWindow];
-    
-    if (newWindow) {
-        [self addLayoutGuideObserversIfNeeded];
-    } else {
-        [self removeLayoutGuideObserversIfNeeded];
-    }
-}
-
-- (void)addLayoutGuideObserversIfNeeded
-{
-    UIViewController *viewController = self.viewControllerForLayoutGuides;
-    BOOL useLayoutGuides = viewController.view && viewController.automaticallyAdjustsScrollViewInsets;
-    
-    if (useLayoutGuides && viewController.topLayoutGuide && !_isObservingTopLayoutGuide) {
-        [(NSObject *)viewController.topLayoutGuide addObserver:self forKeyPath:@"bounds" options:0 context:(void *)&MGLLayoutGuidesUpdatedContext];
-        _isObservingTopLayoutGuide = YES;
-    }
-    
-    if (useLayoutGuides && viewController.bottomLayoutGuide && !_isObservingBottomLayoutGuide) {
-        [(NSObject *)viewController.bottomLayoutGuide addObserver:self forKeyPath:@"bounds" options:0 context:(void *)&MGLLayoutGuidesUpdatedContext];
-        _isObservingBottomLayoutGuide = YES;
-    }
-}
-
-- (void)removeLayoutGuideObserversIfNeeded
-{
-    UIViewController *viewController = self.viewControllerForLayoutGuides;
-    
-    if (_isObservingTopLayoutGuide) {
-        [(NSObject *)viewController.topLayoutGuide removeObserver:self forKeyPath:@"bounds" context:(void *)&MGLLayoutGuidesUpdatedContext];
```

Comment:
Nit: `MGLLayoutGuidesUpdatedContext` can be removed.

Reply:
🙇 fixed in https://github.com/mapbox/mapbox-gl-native/pull/9995/commits/0a19195f6ad01c165b4242d7f72dcc7b07b0d9ca